Main-bigint/Main-bigint/main.cpp

```diff
@@ -0,0 +1,90 @@
+#include <iostream>
+#include <fstream>
+
+#include <math.h>
+#include <string>
+#include <vector>
+#include <bitset>
+
+#include "bigint.h"
+#include "calculate.h"
+
+void rwConsole()
+{
+    // Read and write value with console
+    BigInteger bigInt2 = readConsole();
+    writeConsole(bigInt2);
+}
+
+void rwTextFile(const std::string &inputFile, const std::string &outputFile)
+{
+    std::ifstream textFile(inputFile, std::ios::in);
+    std::string line;
+    if (textFile.is_open())
+        getline(textFile, line);
+    else
+    {
+        std::cout << "rwTextFile(): Cannot open file/ File does not exist.\n";
+        return;
+    }
+
+    int numbersOfBigInt = stoi(line);
```

Comment:
What happens if the first line contains a big integer?

Reply:
Should it throw an error? If I have `array[i]` where `i` is a BigInteger, will it cause overflow?